commands/general/help.js

```diff
@@ -16,13 +16,17 @@ class Help extends Command {
 
   async run(message, [type, page], level) {
 
+    type ? type = type.toLowerCase() : undefined;
```

Comment:
Assigning with a ternary is a bad practice so `type = type ? type.toLowerCase() : null`
Use `null` instead (More style than anything tho)

Reply:
No, because it needs to be ``undefined``, not ``null``.

Reply:
@MrJacz is right.

```javascript
if (type) type = type.toLowerCase();
```